src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -144,20 +144,3 @@ CREATE TABLE warehouse (
 --add constraints and indexes
 CREATE INDEX idx_customer_name ON customer (c_w_id,c_d_id,c_last,c_first);
 CREATE INDEX idx_order ON oorder (o_w_id,o_d_id,o_c_id,o_id);
--- tpcc-mysql create two indexes for the foreign key constraints, Is it really necessary?
--- CREATE INDEX FKEY_STOCK_2 ON STOCK (S_I_ID);
--- CREATE INDEX FKEY_ORDER_LINE_2 ON ORDER_LINE (OL_SUPPLY_W_ID,OL_I_ID);
-
---add 'ON DELETE CASCADE'  to clear table work correctly
-
-ALTER TABLE district  ADD CONSTRAINT fkey_district_1 FOREIGN KEY(d_w_id) REFERENCES warehouse(w_id) ON DELETE CASCADE;
```

Comment:
Can we add a comment block here about the fact that all foreign key constraints are now loaded as deferred constraints?

Also, can these be loaded as non-deferred constraints? Would be good to call out those details here. Otherwise, someone looking at the schema file would think there are no FKs.

Reply:
Sure.